src/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import FlipCounter from "./FlipCounter";
+
+import "./App.css";
+class App extends Component {
+  render() {
+
+    return (
+      <div id="body">
+        <FlipCounter />
+</div>
```

Comment:
The file is not formatted with prettier.

Let's do something to make your code always beautiful.

1. Install [prettier](https://www.npmjs.com/package/prettier) to your dev dependencies
1. Create an npm script (inside scripts object inside package.json) to run prettier across all the project. The name of the script should be "prettier", the value: `prettier --write \"src/**/*.ts*\"`
1. Commit
1. `npm run prettier` to run prettier across all project
1. Commit
1. Now you need to make sure, that prettier will be run across all files on each commit. Install [husky](https://www.npmjs.com/package/husky) and [lint-staged]() to your dev dependencies
1. Create a following `.lintstagedrc` file:
```
{
  "*.ts": ["prettier --write", "git add"],
  "*.tsx": [
    "tslint --fix",
    "prettier --write",
    "git add"
    ],
}
```
1. Add husky config to package.json:

```
"husky": {
    "hooks": {
      "pre-commit": "lint-staged",
    }
  }
```

1. Commit
1. Change any .ts file in the project (just delete the line break, for example) and make the formatting ugly
1. Commit, you will see that husky launches lint-staged, which runs prettier across the file and adds it to git repo